index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="main-section">
+  <header class="header">
+    <nav class="navbar">
+    <a class="logo" href="#">My Logo</a>
```

Comment:
- [ ] [OPTIONAL] I think it would be nice if the logo's font matched the font-family in the design file. What do you think 🤔?